# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/credentialmanagement/identity/tests/mochitest/mochitest.toml
# Commit: ccd96b99aea2
# Full Hash: ccd96b99aea28e8214e585b864d3fb2245bac142
# Author: Atila Butkovits <abutkovits@mozilla.com>
# Date: 2025-06-07 08:31:23
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing failures complaining about fedcm-no-registered-idps.https.html.
#   
#   This reverts commit ba33457e0cbac24a8b7d963c45138c62e92a74ae.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r 4bfadfc33207 -r ccd96b99aea2 dom/credentialmanagement/identity/tests/mochitest/mochitest.toml
--- a/dom/credentialmanagement/identity/tests/mochitest/mochitest.toml	Fri Jun 06 21:07:41 2025 +0300
+++ b/dom/credentialmanagement/identity/tests/mochitest/mochitest.toml	Fri Jun 06 21:35:18 2025 +0300
@@ -1,6 +1,7 @@
 [DEFAULT]
 prefs = [
   "dom.security.credentialmanagement.identity.enabled=true",
+  "dom.security.credentialmanagement.identity.heavyweight.enabled=true",
   "dom.security.credentialmanagement.identity.select_first_in_ui_lists=true",
   "dom.security.credentialmanagement.identity.reject_delay.enabled=false",
   "privacy.antitracking.enableWebcompat=false", # disables opener heuristic